_layouts/plugins/katello/documentation.html

```diff
@@ -1,6 +1,6 @@
 ---
 pagename: Plugin Manuals
-versions: [{name: 3.6, displays: '3.6'}, {name: 3.5, display: '3.5'}, {name: 3.4, display: '3.4'}, {name: 3.3, display: '3.3'}, {name: 3.2, 'display': '3.2'}, {name: 3.1, display: '3.1'}, {name: 3.0, display: '3.0'}, {name: 2.4, display: '2.4'}, {name: 'nightly', display: 'nightly'}]
+versions: [{name: 3.7, display: '3.7'}, {name: 3.6, displays: '3.6'}, {name: 3.5, display: '3.5'}, {name: 3.4, display: '3.4'}, {name: 3.3, display: '3.3'}, {name: 3.2, 'display': '3.2'}, {name: 3.1, display: '3.1'}, {name: 3.0, display: '3.0'}, {name: 2.4, display: '2.4'}, {name: 'nightly', display: 'nightly'}]
```

Comment:
You'll want to update this to `display: '3.7-RC'` so folks don't think it's released yet.